google-cloud-tasks/lib/google/cloud/tasks/v2beta2/cloud_tasks_client.rb

```diff
@@ -448,9 +448,9 @@ def initialize \
           #   object.
           # @raise [Google::Gax::GaxError] if the RPC is aborted.
           # @example
-          #   require "google/cloud/tasks/v2beta2"
+          #   require "google/cloud/tasks"
           #
-          #   cloud_tasks_client = Google::Cloud::Tasks::V2beta2.new
+          #   cloud_tasks_client = Google::Cloud::Tasks::V2beta2.new(version: :V2beta2)
```

Comment:
Huh, this doesn't seem right...

Reply:
Agreed. It should be either `Google::Cloud::Tasks::V2beta2.new` or `Google::Cloud::Tasks.new(version: :V2beta2)`.

Reply:
@jbolinger FYI. Is this something you can help with?

Reply:
Filed https://github.com/googleapis/gapic-generator/issues/2143

Reply:
FWIW, I think it would be better if these version suymbols were all lower-case, eg. `: v2beta2` vs. `: V2beta2`.

Reply:
Agreed. I made a note of that in googleapis/gapic-generator#2143